custom/conf/app.ini.sample

```diff
@@ -178,6 +178,7 @@ MINIMUM_KEY_SIZE_CHECK = false
 ; Disable CDN even in "prod" mode
 OFFLINE_MODE = false
 DISABLE_ROUTER_LOG = false
+MACARON_LOG_LEVEL = console
```

Comment:
How about put this option under `[log]`?

Reply:
OK I've moved to a simpler option of REDIRECT_MACARON_LOG and placed it in the log section.